### ISSUE TITLE
NullPointerException on building insertion variant with germline calling with 1.5.7

### ISSUE THREAD

**Issue by chapmanb** [AstraZeneca-NGS/VarDictJava#166]
Polina and all;
We ran into a ssue with 1.5.7 during germline calling when creating the variant output for an apparent insertion:
```
Exception in thread "main" java.lang.NullPointerException
        at com.astrazeneca.vardict.modules.ToVarsBuilder.toVars(ToVarsBuilder.java:238)
        at com.astrazeneca.vardict.VarDict.vardictNotParallel(VarDict.java:365)
        at com.astrazeneca.vardict.VarDict.nonAmpVardict(VarDict.java:312)
        at com.astrazeneca.vardict.VarDict.start(VarDict.java:72)
        at com.astrazeneca.vardict.Main.run(Main.java:153)
        at com.astrazeneca.vardict.Main.main(Main.java:31)
```
This is a minimal self-contained test case that reproduces the problem:

https://s3.amazonaws.com/chapmanb/testcases/vardict_germline_null.tar.gz 

Thanks so much for looking at it and please let me know I can provide any other information.

**Comment by PolinaBevad** [comment]
@chapmanb, hello!
Thank you for reporting this!

I checked it, the problem is reproduced on both Java and Perl versions, so I will prepare PRs for them. It appears while the insertions are preparing for the output. 
Thank you for preparing the test case!

**Comment by PolinaBevad** [comment]
@chapmanb , hello!
I will close this issue, it was fixed in Java and Perl in 1.5.8 version.
Thank you!
